es6-template/services/helloworld/config/testing.js

```diff
@@ -0,0 +1,21 @@
+const baseConf = require('./development')
+    , ConfigManager = require('yessql-core/utils/ConfigManager')
+    , MemorySessionStore = require('yessql-features/token-session').MemorySessionStore;
+
+baseConf.profile = 'testing';
+baseConf.database.database = 'helloworld-test';
+
+// Disable pooling for tests. This is needed because we usually drop the database multiple times
+// during a test session and we don't want any dangling connections to die because of that.
+baseConf.database.maxConnectionPoolSize = 1;
```

Comment:
I'm not sure if this is needed anymore, since we usually just truncate DB, not drop it.